### PR TITLE
Completed Home Page

### PR DESCRIPTION
**Pull request by Giviko31**
Added buttons in html ( sign and login )
and style it  .
Created two Paragraph head and bottom and added home image . Created styles for this two section.

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -38,4 +38,70 @@
...
+.title_bottom p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 36px;
+    line-height: 54px;
+    color: #4F4F4F;
+}
+
+.home_image img{
+    margin: 0 0 0 170px;
```
Comment (AlexZviadadze) [inline styles/style.css:103]: good job

File: index.html [hunk 2]
```diff
@@ -13,11 +13,32 @@
...
-            
+            <div class="title_head">
+                <p>
+                    bare minimum analytics for 
+                    privacy consious websites
+                </p>
+            </div>
+
+            <div class="home_image">
+                <img src="images/image.svg" alt="Image">
```
Comment (AlexZviadadze) [inline index.html:32]: I like  your class name

**Review by AlexZviadadze — COMMENTED** [review]
I like  your  code it's good

File: index.html [hunk 3]
```diff
@@ -13,11 +13,32 @@
             <div class="box_header">
                 <a href="index.html"><img src="images/logo.svg" alt="logo">Simple Analytics</a>
             </div>
+
```
Comment (Vanoo123) [inline index.html:16]: Good Buttons mate <3

**Review by Vanoo123 — COMMENTED** [review]
Nice bro good vision

**Review by AlexZviadadze — APPROVED** [review]

**Pull request merged**